msal/tests/Test.MSAL.NET.Unit.net45/CoreTests/Telemetry/TelemetryHelperTests.cs

```diff
@@ -35,140 +35,117 @@ namespace Test.Microsoft.Identity.Core.Unit.Telemetry
         public void TestTelemetryHelperWithFlush()
         {
-            using (_telemetryManager.CreateTelemetryHelperEx(requestId, _startEvent, shouldFlush: true))
+            using (_telemetryManager.CreateTelemetryHelper(RequestId, ClientId, _startEvent, shouldFlush: true))
             {
             }
 
-            ValidateResults(_telem, requestId, _startEvent, _startEvent, true);
+            ValidateResults(_testReceiver, RequestId, ClientId, _startEvent, _startEvent, true);
         }
 
         [TestMethod]
         [TestCategory("TelemetryHelperTests")]
         public void TestTelemetryHelperWithDifferentStopStartEvents()
         {
-            using (_telemetryManager.CreateTelemetryHelperEx(requestId, _startEvent, eventToEnd: _stopEvent))
+            using (_telemetryManager.CreateTelemetryHelper(RequestId, ClientId, _startEvent, eventToEnd: _stopEvent))
             {
             }
 
-            ValidateResults(_telem, requestId, _startEvent, _stopEvent, false);
+            ValidateResults(_testReceiver, RequestId, ClientId, _startEvent, _stopEvent, false);
         }
 
         [TestMethod]
         [TestCategory("TelemetryHelperTests")]
         public void TestTelemetryHelperWithDifferentStopStartEventsWithFlush()
         {
-            using (_telemetryManager.CreateTelemetryHelperEx(requestId, _startEvent, eventToEnd: _stopEvent, shouldFlush: true))
+            using (_telemetryManager.CreateTelemetryHelper(RequestId, ClientId, _startEvent, eventToEnd: _stopEvent, shouldFlush: true))
             {
             }
 
-            ValidateResults(_telem, requestId, _startEvent, _stopEvent, true);
+            ValidateResults(_testReceiver, RequestId, ClientId, _startEvent, _stopEvent, true);
         }
 
         private void ValidateResults(
-            _TestTelem telem,
+            _TestReceiver _testReceiver,
             string expectedRequestId,
+            string expectedClientId,
             _TestEvent expectedStartEvent,
             _TestEvent expectedStopEvent,
             bool shouldFlush)
         {
-            if (shouldFlush)
-            {
-                Assert.AreEqual(1, telem.NumFlushCalls);
-                Assert.AreEqual(expectedRequestId, telem.LastFlushEventRequestId);
-            }
-            else
-            {
-                Assert.AreEqual(0, telem.NumFlushCalls);
-                Assert.AreEqual(string.Empty, telem.LastFlushEventRequestId);
-            }
-
-            Assert.AreEqual(1, telem.NumStartEventCalls);
-            Assert.AreEqual(1, telem.NumStopEventCalls);
-
-            Assert.AreEqual(expectedRequestId, telem.LastStartEventRequestId);
-            Assert.AreEqual(expectedRequestId, telem.LastStopEventRequestId);
-
-            Assert.AreEqual(expectedStartEvent, telem.LastEventToStart);
-            Assert.AreEqual(expectedStopEvent, telem.LastEventToStop);
+            // TODO: implement this to check for proper events...
```

Comment:
Dead code?

Reply:
Incomplete code.  I sent out the PR initially to check for passage of unit tests and get initial feedback.  I still need to restore some sanity to these tests before completion...